scripts/pin_the_slow_way.py

```diff
@@ -29,22 +29,22 @@
 
 
 pinned = {
-          'boost': 'boost 1.60.*',
-          'bzip2': 'bzip2 1.0.*',
-          'fontconfig': 'fontconfig 2.11.*',
+          'boost': 'boost 1.61*',
```

Comment:
Right that is clear. In that case, I think you are correct. Mainly I'm worried about the `.` for things like `2.1*`. If we build with `2.1.0` and then rapidly reach `2.10.0`, we could have problems. We should hopefully stay on top of this.

In most cases, I think this is not too big of a deal. For instance, Boost (I know) uses two digits for the minor version due to C++ constraints when exposing the version. So, we have no risk of hitting Boost `1.610.0` for example.

Still this is an easy to implement measure that costs us very little. Also there is always the potential for a solver bug or other problems. This simple measure could save us a later fiasco.